renovate.json

```diff
@@ -1,5 +1,20 @@
 {
+  "timezone": "Europe/Berlin",
   "extends": [
-    "config:base"
+    "config:base",
+    ":maintainLockFilesWeekly",
+    ":prNotPending"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 4pm on Tuesday"
```

Comment:
Could we move this to end of Monday or early Tuesday, so we have enough time to deal with potential breakings?

Reply:
I think the default is on Monday morning (European time), which I find convenient.

Reply:
early Tuesday makes sense, that way we can make a new release of dispensary on Monday ideally.

Fixed in 7f775ffb